user/enterprise/platform-tips.md

```diff
@@ -309,6 +309,16 @@ Additionally, please the following command to clean up all Docker build images:
 $ sudo docker images | grep travis | awk '{print $3}' | xargs sudo docker rmi -f
 ```
 
+## Integrate Travis CI Enterprise into your monitoring
+
+To implement a simple uptime check for Travis CI Enterprise, you can query the `/api/uptime` endpoint from your instance.
```

Comment:
It is best to avoid using words like "simple" here.

Reply:
Also, this doesn't check "uptime" it just checks "upness" :-p

How about 

> To check if your Travis CI Enterprise (what? installation? cluster? Instance?) is up and running, query the `/api/uptime` endpoint of your instance.

We do mean of, not from, right?

Reply:
I've just noticed the endpoint is called uptime. :/

Reply:
Yeah, it's called `uptime` but returns a simple `OK` if everything checks out.